test/e2e/ig_latency.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"runtime"
+	"time"
+)
+
+func TestWithImportGroupLatency(
+	apiName string, opt *TestOption, testSuite func(*TestOption)) error {
+	durations := map[bool]float64{} // Map: useImportGroup -> duration in seconds.
+
+	for _, useImportGroup := range []bool{true, false} {
```

Comment:
For reasonable performance evals, to avoid noise, it's generally a good idea to run a few times and report the individual numbers or mean/variance ?